fearless/Common/Model/ChainRegistry/ChainModel.swift

```diff
@@ -298,7 +298,11 @@ extension ChainModel {
     }
 
     func polkascanAddressURL(_ address: String) -> URL? {
-        URL(string: "https://explorer.polkascan.io/\(name.lowercased())/account/\(address)")
+        if isPolkadotOrKusama {
```

Comment:
use `func explorerUrl(for value: String, type: ChainModel.SubscanType) -> URL?` for return optional URL